eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -68,11 +68,12 @@ stages:
         name: ${{ variables.defaultPoolName }}
         demands: ${{ variables.defaultPoolDemands }}
       container: ${{ parameters.centOSStream8Container }}
-      bootstrapPrep: false            # 🚫
-      enablePoison: false             # 🚫
-      excludeSdkContentTests: true    # ✅
-      excludeOmniSharpTests: true     # ✅
-      runOnline: true                 # ✅
+      bootstrapPrep: false          # 🚫
+      enablePoison: false           # 🚫
+      excludeSdkContentTests: true  # ✅
+      excludeOmniSharpTests: true   # ✅
+      runOnline: true               # ✅
+      buildFromArchive: false       # 🚫
 
   - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
     - template: ../jobs/vmr-build.yml
```

Comment:
I was expecting a change to the jobs template.  Was that possible not committed?

Reply:
Ah, shoot. Yes that's what happened